lib/WorkerHandler.js

```diff
@@ -116,6 +129,12 @@ function WorkerHandler(script) {
     me.processing = {};
   }
 
+  // send all queued request to worker
+  function dispatchQueuedRequests()
+  {
+    me.request_queue = me.request_queue.filter(me.worker.send.bind(me.worker));
```

Comment:
Methods like `forEach`, `map`, and `filter` are all functional. I understand what you did and why but the way you used `filter` is isn't really wat it is intended for: sending messages inside a `filter` action is a side effect which is the opposite of a functional approach, and these send messages don't return anything so it's misleading to filter based on a function which always returns `undefined`. Anyway, thanks for changing it!